glass: Basic support of i18n (#233)
aquarist-labs/aquarium: doc/i18n.md

# I18N

The `Glass` project supports internationalization. This document
describes how to extract and process the translations.

## How to become a translator
If you want to become a translator, please create a new account at [Transifex](https://www.transifex.com/)
and join the [Aquarium](https://www.transifex.com/aquarist-labs/aquarium/) project.

## Process translations in `Glass`

### Prerequisites
To be able to pull the translations from Transifex, `Python 3` must be
installed on your system.

### Extract translations
To extract the i18n messages from the HTML templates and the TypeScript

> files just  run the following command in `src/glas`:

Reviewer: ```suggestion
files just run the following command in `src/glass`:
```

```
$ npm run i18n:extract
```
The file `src/glass/src/assets/i18n/aquarium.pot` will be created. This file

> must be uploaded manually at Transifex to update the source file of the

Reviewer: I'm not an expert but `at` feels a bit unusual to me in the context. Maybe -

```suggestion
must be uploaded manually to Transifex in order to update the source file of the
```

?

> must be uploaded  manually ta Transifex to update the source file of the

Reviewer: ```suggestion
must be uploaded manually to Transifex to update the source file of the
```
